open-metadata-implementation/user-interfaces/ui-chassis/ui-chassis-spring/pom.xml

```diff
@@ -326,7 +323,7 @@
                                 <configuration>
                                     <nodeVersion>${node.version}</nodeVersion>
                                     <npmVersion>${npm.version}</npmVersion>
-                                    <workingDirectory>tools</workingDirectory>
+                                    <workingDirectory>../ui-chassis-spring/tools</workingDirectory>
```

Comment:
I'm a little wary of creating files outside the directory tree we are working in. Clean may get confused, as may other tools such as javadoc or scans. I would like to look at this in a little more detail when I get a chance (tomorrow) to check out the code.

Reply:
Seems to work ok. let's close for now.